Shared/Samples/Browse WFS layers/BrowseWFSLayersView.swift

```diff
@@ -0,0 +1,194 @@
+                    }
+                }
+            }
+            .navigationTitle(serviceInfo.title)
+        } else {
+            ProgressView()
+                .navigationTitle("Loading Service")
+                .task {
+                    let service = WFSService(url: serviceURL)
+                    do {
+                        try await withTaskCancellationHandler {
+                            try await service.load()
+                        } onCancel: {
+                            service.cancelLoad()
+                        }
+                        self.service = service
+                    } catch {
+                        serviceLoadError = error
+                    }
+                }
+                .alert("Error", isPresented: $serviceLoadDidFail, presenting: serviceLoadError) { _ in
+                    Button("OK") {
+                        dismiss()
+                    }
+                } message: { error in
+                    Text(String(reflecting: error))
+                }
+        }
+    }
+}
+
+/// A view that displays a layer of a WFS service.
+private struct WFSServiceLayerView: View {
+    /// The metadata of the layer to display.
+    let layerInfo: WFSLayerInfo
+    
+    /// The map that displays the layer.
+    @State private var map = Map(basemapStyle: .arcGISImageryStandard)
+    /// The area of the map to display.
+    @State private var viewpoint: Viewpoint?
+    /// The error if the populate operation failed, otherwise `nil`.
+    @State private var populateError: Error?
+    /// A Boolean value indicating whether a query operation is in progress.
+    @State private var isQuerying = false
+    /// A Boolean value indicating whether the axis order should be swapped.
+    @State private var swapAxisOrder = false
+    
+    var body: some View {
+        MapView(map: map, viewpoint: viewpoint)
+            .onViewpointChanged(kind: .boundingGeometry) { newViewpoint in
+                viewpoint = newViewpoint
+            }
+            .onAppear {
+                if let extent = layerInfo.extent {
+                    viewpoint = Viewpoint(boundingGeometry: extent)
+                }
+            }
+            .task(id: swapAxisOrder) {
+                map.removeAllOperationalLayers()
+                let featureTable = WFSFeatureTable(layerInfo: layerInfo)
+                // Sets the feature request mode to 'manualCache'. In this mode,
+                // you must manually populate the table - panning and zooming
+                // won't request features automatically.
+                featureTable.featureRequestMode = .manualCache
+                featureTable.axisOrder = if swapAxisOrder {
+                    .swap
+                } else {
+                    .noSwap
+                }
+                do {
+                    isQuerying = true
+                    _ = try await featureTable.populateFromService(using: nil, clearCache: false, outFields: [])
```

Comment:
Should `clearCache` be `true` here? `false` seems to work fine for me, but I also don't have a swapped service to test if the order change has any effect on the cache.

Reply:
I don't know, but [.NET uses `false`](https://github.com/Esri/arcgis-maps-sdk-dotnet-samples/blob/c1b6b68a45b2210981eb722428f2d394c2fdc2e9/src/MAUI/Maui.Samples/Samples/Layers/BrowseWfsLayers/BrowseWfsLayers.xaml.cs#L117). I just copied what they did.